docs/blog/2020-06-18-0.0.10-released.md

```diff
@@ -217,11 +217,12 @@ The `<RecoilRoot>` component also has an `initializeState` prop which can be use
 function MyApp() {
   return (
     <RecoilRoot
-      initializeState(({set}) => {
+      initializeState={({ set }) => {
```

Comment:
Thank you for the fix!  Though, our prettier config doesn't place spaces inside destructuring braces.

Reply:
ah sorry, didn't notice. fixed.